src/v2i-hub/MessageReceiverPlugin/src/MessageReceiverPlugin.cpp

```diff
@@ -263,6 +263,7 @@ void MessageReceiverPlugin::OnMessageReceived(routeable_message &msg)
 								}
 
 								sendMsg = encode(encodedBsm, bsm);
+								msgPSID = 0x20;
```

Comment:
Do different messages have different msgPSID? Does a PSID ever change for a message type? I am not sure if we want to hardcode these  values for different msgPSID?

Reply:
These two I added were under the cases for BSMs and SRMs that are detected by the plugin. Others are kept at null. 
If we want to add more messages to be routable, we can think about where to add their PSIDs. Right now it's added within the specific plugins that generate the messages.

Reply:
Do we have a enum or struct to consolidate all the PSID? Then use it here in this class? It might be easy to read.

Reply:
We don't, but we could make one. What do you think about adding it here? https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/src/tmx/TmxApi/tmx/TmxApiMessages.h

Reply:
yep, I think that is a good location

Reply:
@dan-du-car Changes made